pkg/core/pipeline/conditions.go

```diff
@@ -42,22 +42,24 @@ func (p *Pipeline) RunConditions() (globalResult bool, err error) {
 				p.Sources[condition.Config.SourceID].Output +
 				p.Sources[condition.Config.SourceID].Config.Postfix)
 
+		if err != nil {
+			// Show error to end user
+			logrus.Error(err)
+			if condition.Result != result.SUCCESS {
+				globalResult = false
+			}
+		}
 		rpt.Result = condition.Result
 
+		// Update pipeline after each condition run
 		if err == nil {
```

Comment:
Does this check is useful as the opposite is checked l45 now?

Reply:
yes it is: my understanding of the code here (yes, no tests nor comment) to run ALL conditions despite their return status (true/fase + error or not).

I've updated the code with an "else" directive as they are no "early return". It should help for readibility WDYT?

Reply:
I like it jetter indeed :)